README.md

```diff
@@ -70,6 +70,9 @@ The best way to subscribe to OpenTelemetry events is to join the [calendar updat
 Group](https://groups.google.com/forum/#!forum/opentelemetry-calendar). This will invite you to every OpenTelemetry call and correctly block time for
 accepted events, and it will keep your calendar in sync with new and updated events.
 
+(Internal Note: When creating or updating a calendar event in the `OpenTelemetry Public Calendar`, please ensure that `opentelemetry-calendar@googlegroups.com`
+is added as an attendee to the meeting invite.  This is what allows updates to the calendar to be synchronized to other attendees.)
```

Comment:
This is already mentioned in https://github.com/open-telemetry/community/blob/main/docs/how-to-handle-public-calendar.md#create-the-meeting but as you brought up many existing meetings lack the group, it probably makes sense to mention it here as well.